src/newHome/homeInfoTiles/TilesContainer.tsx

```diff
@@ -7,8 +8,13 @@ import { getTilesData } from "./tileData";
 
 export const HomeTilesContainer: React.FC = () => {
   const intl = useIntl();
+  const posthog = usePostHog();
 
-  const tiles = getTilesData({ intl });
+  const handleTileButtonClick = (tileId: string) => {
+    posthog.capture(`home_page:${tileId}_click`);
```

Comment:
it will be hard to build reports with it, track payload separatelly:
```
posthog.capture('home_tile_click', { tile_id: tileId });
```

By doing this I can:
- track all tiles click easily
- aggregate by certain tile

Your solution will probably also work, but we would have to do some regexing.